CHANGELOG.md

```diff
@@ -1,6 +1,68 @@
-## 2.7.0 (on-going)
-
-> ????/??/??
+## 2.8.0 (on-going)
```

Comment:
Please respect the style of the rest of the document by using whitespaces between lines starting with a sharp (`#`) and indenting the dashes (`-`) for bullet lists, even for the first level. Mixing styles in such a document will lead to confusion and it’s better to keep things clean. 🙂

Reply:
Same comment for the endings of the sentences in bullet lists: the whole document uses `periods`, not `commas`; please switch to periods.